### ISSUE TITLE
CDATA filed not rendered

### ISSUE THREAD

**Issue by iloahz** [storybookjs/react-inspector#33]
result in console:
![screen shot 2017-06-26 at 5 30 12 pm](https://user-images.githubusercontent.com/849273/27533249-7e188556-5a95-11e7-9310-db3ce02267ad.png)

result in web page using react-inspector:
![screen shot 2017-06-26 at 5 30 23 pm](https://user-images.githubusercontent.com/849273/27533280-9304a652-5a95-11e7-9d09-2179ba480a53.png)

code looks like this:
```
        let parser = new DOMParser();
        d = parser.parseFromString("<xml><a><![CDATA[http://fdsafsd.com/fdsa?abc=1]]></a></xml>", "application/xml");
        return (
            <Inspector expandLevel={99} data={d} />
        );
```
can you help take a look?

**Comment by iloahz** [comment]
created a pr at https://github.com/xyc/react-inspector/pull/34

**Comment by ndelangen** [comment]
Released with 2.1.0
